src/Results/Results.test.tsx

```diff
@@ -24,19 +27,71 @@ describe('Results', () => {
 				'continent': 'North America',
 				'label': 'Miami, FL, USA'
 			}
+		],
+		mockedWorldResults = [
```

Comment:
I think we can put all of our minds together and crush through these async tests in no time. Great work